packages/components/addon/components/hds/disclosure-primitive/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { schedule } from '@ember/runloop';
+
+export default class HdsDisclosurePrimitiveComponent extends Component {
+  @tracked isOpen; // notice: if in the future we need to add a "@isOpen" prop to control the status from outside (eg to have the DisclosurePrimitive opened on render) just add  "this.args.isOpen" here to initalize the variable
+  @tracked toggleRef;
+
+  @action
+  didInsert(element) {
+    this.element = element;
+  }
```

Comment:
I don't think this is needed anymore (it's used in the `onFocusOut` method, but now it's been deleted in this primitive)

```suggestion
```